core/vm/evm.go

```diff
@@ -645,9 +645,9 @@ func (evm *EVM) CreateETX(toAddr common.Address, fromAddr common.Address, gas ui
 // encourage processing at the destination.
 func calcEtxFeeMultiplier(fromAddr, toAddr common.Address) *big.Int {
 	confirmationCtx := fromAddr.Location().CommonDom(*toAddr.Location()).Context()
-	multiplier := big.NewInt(common.NumZonesInRegion)
+	multiplier := big.NewInt(common.MaxRegions) //  TODO:  calculation  should be based on the current  expansion  number
```

Comment:
Is this for this PR?

Reply:
maybe not, we will come back when we have to go through some ofn these parameters